src/main/environment/common_ci.properties

```diff
@@ -67,6 +67,7 @@ tcSpeclistWL=7
 
 ### Redis IP
 spring.redis.host=localhost
+jwt.secret=@JWT_SECRET_KEY@
```

Comment:
use env.JWT_SECRET_KEY
check how it's added in other places.

Reply:
fixed